client/js/views/GameView.vue

```diff
@@ -340,21 +307,24 @@
         @targetedOneOff="beginTargeting($event)"
         @cancel="clearSelection"
       />
+      <game-dialogs
+        :clear-selection="clearSelection"
+        :game="game"
+        :handle-error="handleError"
+        :resolving-seven="resolvingSeven"
```

Comment:
Because both these methods directly mutate data inside the gameView, I think it might be more Vue-esque to bind these as event handlers that get called by the GameView to avoid having the GameDialogs directly updating GameView data. What do you think?

Reply:
Yep, moved to emitters/events in the latest push.